site/content/en/docs/Tutorials/external_tutorials.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "External Tutorials On Minikube"
+linkTitle: "External Tutorials On Minikube"
+weight: 1
+date: 2019-10-31
+description: >
+  Adding link to external tutorials based on minikube
+---
+
+External tutorials about minikube:
+
+https://quarkus.io/guides/deploying-to-kubernetes
```

Comment:
I'm OK with the concept, but this tutorial does not do enough with minikube to really call it a tutorial. It does not focus on minikube, and barely fits with the idea that a tutorials are  "complete worked examples made up of multiple tasks that guide the user through a relatively simple but realistic scenario".

Can you find another external example to use?

Reply:
sure